boot/freeldr/freeldr/arch/i386/drvmap.S

```diff
@@ -56,8 +55,8 @@ Int13Handler:
 
 GetMappedDriveNumberLoop:
 
-    /* Get the next drive number in the list */
-    lods ax, cs:[si]
+    /* Get the next drive number in the list (store it in AX) */
```

Comment:
Mentioning the destination register seemed to be a GCC extension, but since AX is also the default in x86 for lods I just drop it and use the standard syntax that works in MSVC.

Reply:
It even can be just "lodsw"